src/Test.FunctionalTests.BinSkim.Driver/AnalyzeCommandTest.cs

```diff
@@ -1,15 +1,77 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.BinaryParsers;
+using Microsoft.CodeAnalysis.IL;
+using Microsoft.CodeAnalysis.Sarif;
+
+using Moq;
+
+using Newtonsoft.Json;
+
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.BinSkim.Driver
 {
     public class AnalyzeCommandTest
     {
+        private const string ToolName = "binskim";
+        private readonly string SarifLogV1 = $@"{{
+  ""$schema"": ""http://json.schemastore.org/sarif-1.0.0"",
+  ""version"": ""1.0.0"",
+  ""runs"": [
+    {{
+      ""tool"": {{
+        ""name"": ""{ToolName}""
+      }},
+      ""results"": []
+    }}
+  ]
+}}";
```

Comment:
UBERNIT: not sure if some align will look better :)

Reply:
I like to put the opening {{ on a separate line to accomplish this

Reply:
removed the string and created a file containing it.

Reply:
test